krnlmon/CMakeLists.txt

```diff
@@ -4,8 +4,6 @@
 # SPDX-License-Identifier: Apache 2.0
 #
 
-cmake_minimum_required(VERSION 3.15)
```

Comment:
Don't we need a `cmake_minimum_required` in krnlmon as we do with the ovs/CMakeLists.txt and stratum/CMakelists, since they are submodules?

Reply:
The `cmake_minimum_required()` for krnlmon is in `krnlmon/krnlmon/CMakeLists.txt`, which is where the `project()` command is. It's the top-level directory in the `krnlmon` repository.

I couldn't do this for `stratum` because cmake support stops at repository boundary. This will change when we move cmake support into the Stratum repository (Stratum PR https://github.com/ipdk-io/stratum-dev/pull/77 and networking-recipe PR https://github.com/ipdk-io/networking-recipe/pull/156).